### DISCUSSION TITLE
does air gapped setup require an account?

### DISCUSSION THREAD

**Discussion by bluebrown** [datreeio/datree#808] [Q&A]
Hi, I am trying to configure air gapped setup. But datree fails with the following warning:

```
to use --policy-config flag you must first enable policy-as-code mode: https://hub.datree.io/policy-as-code
```

When I check this documentation, it links me to some kind of dashboard.

> 1. Enable Policy-as-code (PaC) mode
On the [Settings page](https://app.datree.io/settings), toggle on the Policy-as-code switch.

Is there not other way to get this working? When I don't run in air gapped mode, I also don't need to sign up somewhere.

**Comment by eyarz** [comment]
Hi @bluebrown 
You don't need to signup when using Datree in an air-gapped environment. 
The warning you see should only appear if your machine has internet access.
Can you check if this is the case? If the machine doesn't have internet access, that can be a bug in how Datree's CLI checks it.

**Reply by bluebrown** [reply]
Hi, thank you, my machine has internet access because I am evaluating this tool at the moment. But I think, if I tell it to work offline, it should not question my decision and check if I maybe still have internet access.

**Reply by eyarz** [reply]
No one is perfect :)
you're more than welcome to open an issue or a submit a PR to fix that.

**Reply by bluebrown** [reply]
Sounds good. I will make a fork and take a look.
